src/components/PopupWithForm.js

```diff
@@ -35,7 +34,7 @@ export default class PopupWithForm extends Popup {
     formInputFields.forEach((input) => {
       data[input.name] = input.value;
     });
-    console.log(data);
+
```

Comment:
Needs correction: in the submit handler you should close the popup only after the successful api response in `then` block in index.js, not here

Reply:
Done